src/test/java/core/basesyntax/service/impl/TransactionServiceImplTest.java

```diff
@@ -0,0 +1,124 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.OperationStrategy;
+import core.basesyntax.strategy.BalanceOperationHandler;
+import core.basesyntax.strategy.OperationHandler;
+import core.basesyntax.strategy.PurchaseOperationHandler;
+import core.basesyntax.strategy.ReturnOperationHandler;
+import core.basesyntax.strategy.SupplyOperationHandler;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class TransactionServiceImplTest {
+    private static final Fruit FRUIT_BANANA = new Fruit("banana");
+    private static final Fruit FRUIT_APPLE = new Fruit("apple");
+    private static final int DEFAULT_QUANTITY = 111;
+    private static final Map<Fruit, Integer> storage = Storage.storage;
+    private static TransactionServiceImpl transactionService;
+    private static Map<FruitTransaction.Operation, OperationHandler> operationHandlerMap;
+    private static OperationStrategy strategy;
```

Comment:
should be local variables

Reply:
Moved to local variables.
+Corrected in new commit.